packages/forklift-console-plugin/src/modules/Plans/views/details/tabs/Mappings/PlanMappingsSection.tsx

```diff
@@ -648,31 +683,35 @@ export const PlanMappingsSection: React.FC<PlanMappingsSectionProps> = ({
   return state.edit ? (
     // Edit mode
     <>
-      <Flex>
-        <FlexItem>
-          <Button
-            variant="primary"
-            onClick={onUpdate}
-            isDisabled={!state.dataChanged}
-            isLoading={isLoading}
-          >
-            {t('Update mappings')}
-          </Button>
-        </FlexItem>
-        <FlexItem>
-          <Button variant="secondary" onClick={onCancel}>
-            {t('Cancel')}
-          </Button>
-        </FlexItem>
-      </Flex>
-      <HelperText className="forklift-section-plan-helper-text">
-        <HelperTextItem variant="indeterminate">
-          {t(
-            'Click the update mappings button to save your changes, button is disabled until a change is detected.',
-          )}
-        </HelperTextItem>
-      </HelperText>
-      <Divider />
+      {!state.editAction && (
+        <>
+          <Flex>
+            <FlexItem>
+              <Button
+                variant="primary"
+                onClick={onUpdate}
+                isDisabled={!state.dataChanged}
+                isLoading={isLoading}
+              >
+                {t('Update mappings')}
+              </Button>
+            </FlexItem>
+            <FlexItem>
+              <Button variant="secondary" onClick={onCancel}>
+                {t('Cancel')}
+              </Button>
+            </FlexItem>
+          </Flex>
+          <HelperText className="forklift-section-plan-helper-text">
+            <HelperTextItem variant="indeterminate">
+              {t(
+                'Click the update mappings button to save your changes, button is disabled until a change is detected.',
+              )}
+            </HelperTextItem>
+          </HelperText>
+          <Divider />
+        </>
+      )}
```

Comment:
Please open a task under the epic of tech debt to break this file

Reply:
OK

Reply:
https://issues.redhat.com/browse/MTV-1968